examples/modules/async-number/binding.gyp

```diff
@@ -0,0 +1,39 @@
+{
+  "variables": {
+    "napajs_root": "<!(node -e \" \
+      var path = require('path'); \
+      process.stdout.write(path.resolve(path.dirname(require.resolve('napajs')), '..')); \
```

Comment:
To get napajs_inc and napajs_lib, we have the below 4 options:

### 1.	'bin' in napajs package.json

- a). Napajs provide bin entries of “get_napajs_inc_path” and “get_napajs_lib_path”, and the bin entries deal with relative path within napajs, and prefix/suffix for different platforms.
- b).	In user’s package.jons, call “get_napajs_inc_path” and “get_napajs_lib_path” to get napajs_inc_path and napajs_lib_path
- c).	In user’s package.jons, make the scripts.install “cmake-js -Dnapajs_inc: napajs inc_path -Dnapajs_lib: napajs_lib_path”
### 2.	build-helper from napajs
- a).	expose build-helper from napajs, and the build-helper deal with relative path within napajs, and prefix/suffix for different platforms.
- b).	execute_process(COMMAND node -e “require(‘napajs’).build-helper.resolve_napa_inc()” … NAPAJS_INC …)
- c).	execute_process(COMMAND node -e “require(‘napajs’).build-helper.resolve_napa_lib()” … NAPAJS_LIB …)
### 3.	napa_root by require.resolve('napajs')
- a).	get napa_root by execute_process(COMMAND node -e "var path = require('path'); process.stdout.write(path.resolve(path.dirname(require.resolve('napajs')), '..'));" … NAPAJS_ROOT)
- b).	user deal with the relative path, and prefix/suffix for different platforms.
### 4.	cmake command "find_library"
- a).	find_library deal with prefix/suffix for different platforms.
- b).	user deal with relative path within napajs by listing all possible paths as parameter of find_library. 

Both 1) and 2) needs to update napajs, and we still need to figure out how it works for 1).a.b.c.
After discussion, we decided to go with the implementation of 3) first, then try to move to 1).